module_auto_update/wizard/module_auto_update_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_base_module_upgrade" model="ir.ui.view">
+        <field name="name">Module Upgrade</field>
+        <field name="model">module.auto.update</field>
+        <field name="arch" type="xml">
+            <form string="Auto-Upgrade Modules">
+                <p>Are you sure you want to launch module auto-update?</p>
+                <separator string="Impacted Apps" />
+                <field name="modules_to_update" />
```

Comment:
maybe is better to have a button to show `modules_to_update`

Reply:
Done! Let me know what you think about it, please :)